static-site/content/blog/2024-12-13-auspice-2024.md

```diff
@@ -0,0 +1,120 @@
+---
+author: "The Nextstrain team"
+date: "2024-12-13"
+title: "Notable changes in Auspice"
+sidebarTitle: "Notable changes in Auspice"
+---
+
+The Nextstrain team continues to improve the visualisation capabilities of
+Auspice, including new features, bug fixes and performance improvements. This
+year, we've released 19 versions and wanted to highlight some of the
+significant changes.
+
+<!--
+number of git tags in 2024 (generated by GitHub Copilot):
+git tag --list --format='%(creatordate:short)' | grep '^2024' | wc -l
+-->
+
+## Table of contents
+
+- [View in other platforms](#view-in-other-platforms)
+- [Performance improvements](#performance-improvements)
+- [Convey tip confidence through saturation](#convey-tip-confidence-through-saturation)
+- [Focus on selected](#focus-on-selected)
+- [Filter on all metadata](#filter-on-all-metadata)
+- [Full changelog](#full-changelog)
+
+## View in other platforms
+
+Historically, we have always been strong advocates for interoperability between
+platforms. For many years, Nextstrain.org has supported
+[fetching](https://nextstrain.org/fetch) HTTPS-addressable Auspice JSONs.
+[UShER](https://genome.ucsc.edu/cgi-bin/hgPhyloPlace) uses this feature to link
+to Auspice views for subtrees.
+
+Other platforms have developed a similar approach to access Auspice datasets,
+often using our RESTful API. When viewing datasets in Nextstrain, you can switch
+to supported platforms via the new **VIEW IN OTHER PLATFORMS** button at the
+bottom of the page:
+
+[![View in Other Platforms button](/blog/img/view-in-other-platforms.png)]()
+
+There are currently three supported platforms:
+
+- [Taxonium](https://taxonium.org) (added in v2.53) – scales well to large
+  datasets.
+
+- [MicrobeTrace](https://microbetrace.cdc.gov) by US CDC (added in v2.53)
```

Comment:
"– allows network visualization of the Auspice phylogeny"

Reply:
Done:

https://github.com/nextstrain/nextstrain.org/blob/1f9cc8c088b4deb981bc4f05db87ac166ac1bcf0/static-site/content/blog/2024-12-13-auspice-2024.md#L47-L48